apps/src/util/deprecatedLabWarning.js

```diff
@@ -0,0 +1,8 @@
+export var showDeprecatedLabWarning = function(labName) {
+  if (labName === 'calc' || labName === 'eval') {
```

Comment:
Is this if statement needed at this point? It looks like it will always be true.

Reply:
it will always be true, but I wanted this to be extensible for future deprecated labs. I can remove it if it's overkill

Reply:
I'll adjust this